<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGameDialog</class>
 <widget class="QDialog" name="NewGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Valtaistuinpeli</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="mainDialogLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tervetuloa Valtaistuinpeliin!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="player1Layout">
     <item>
      <widget class="QLabel" name="player1Label">
       <property name="text">
        <string>Pelaaja 1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="player1NameEdit">
       <property name="placeholderText">
        <string>Syötä pelaajanimi tähän</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="player1Check">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Enable Player</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="player2Layout">
     <item>
      <widget class="QLabel" name="player2Label">
       <property name="text">
        <string>Pelaaja 2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="player2NameEdit">
       <property name="placeholderText">
        <string>Syötä pelaajanimi tähän</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="player2Check">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Enable Player</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="player3Layout">
     <item>
      <widget class="QLabel" name="player3Edit">
       <property name="text">
        <string>Pelaaja 3:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="player3NameEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Syötä pelaajanimi tähän</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="player3check">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Enable Player</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="player4Layout">
     <item>
      <widget class="QLabel" name="player4Label">
       <property name="text">
        <string>Pelaaja 4:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="player4NameEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Syötä pelaajanimi tähän</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="player4Check">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Enable Player</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="startGameButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Aloita peli</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>player1NameEdit</tabstop>
  <tabstop>player1Check</tabstop>
  <tabstop>player2NameEdit</tabstop>
  <tabstop>player2Check</tabstop>
  <tabstop>player3NameEdit</tabstop>
  <tabstop>player3check</tabstop>
  <tabstop>player4NameEdit</tabstop>
  <tabstop>player4Check</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>startGameButton</sender>
   <signal>clicked()</signal>
   <receiver>NewGameDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
